charts/flyte-core/values-sandbox.yaml

```diff
@@ -49,6 +49,21 @@ configmap:
     CONFIG_DIR: /etc/flyte/config
     DISABLE_AUTH: "1"
 
+  # -- Task default resources configuration
+  # Refer to the full [structure](https://pkg.go.dev/github.com/lyft/flyteadmin@v0.3.37/pkg/runtime/interfaces#TaskResourceConfiguration).
+  task_resource_defaults:
+    # -- Task default resources parameters
+    task_resources:
+      defaults:
+        cpu: 500m
+        memory: 1Gi
+        storage: 500Mi
+      limits:
+        cpu: 2
+        memory: 4Gi
+        storage: 20Mi
```

Comment:
why is this so low? Do we have to set it?

Reply:
That is the default value in [value.yaml](https://github.com/flyteorg/flyte/blob/f8178c1c093cf641c36c0f00614bd0fbe1581c31/charts/flyte-core/values.yaml#L549). Yup, it's optional. I just removed it to make the config simple.